DemoApp/DemoApp/Views/TableView.xaml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="utf-8"?>
+<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:modus="clr-namespace:Trimble.Modus.Components;assembly=Trimble.Modus.Components"
+             x:Class="DemoApp.TableView"
+             Title="DataGrid">
+  <ScrollView>
+    <VerticalStackLayout HorizontalOptions="Fill">
+      <modus:TMRadioButtonGroup x:Name="RadioGroup"
+                                Orientation="Horizontal"
+                                GroupTitle="Selection Mode"
+                                SelectedIndex="{Binding SelectionModeRadioOption}">
+        <modus:TMRadioButton Text="None" />
+        <modus:TMRadioButton Text="Single" />
+        <modus:TMRadioButton Text="Multiple" />
+      </modus:TMRadioButtonGroup>
+      <HorizontalStackLayout>
+        <Label Text="Show Divider"
+               VerticalOptions="Center" />
+        <Switch IsToggled="{Binding DividerToggled}"
+                VerticalOptions="Center" />
+      </HorizontalStackLayout>
+      <modus:DataGrid ItemsSource="{Binding Users}"
+                      IsSortable="True"
+                      ShowDivider="{Binding DividerToggled, Mode=OneWay}"
+                      SelectionMode="{Binding SelectionMode}"
+                      x:Name="ModusDataGrid">
+        <modus:DataGrid.Columns>
+          <!-- Image column -->
+          <modus:ImageColumn Title="Logo"
+                             PropertyName="Logo"
+                             Width="0.25*" />
+          <!-- Text Column -->
+          <modus:TextColumn Title="Team"
+                            PropertyName="Name"
+                            DescriptionProperty="Description"
+                            Width="0.5*"
+                            HorizontalContentAlignment="Start" />
+          <!-- Custom Column -->
+          <modus:DataGridColumn Title="ID"
+                                PropertyName="ID"
+                                Width="0.25*">
+            <modus:DataGridColumn.CellTemplate>
+              <DataTemplate>
+                <StackLayout VerticalOptions="Center">
+                  <Label Text="{Binding ID}"
+                         Padding="16,0" />
+                </StackLayout>
+              </DataTemplate>
+            </modus:DataGridColumn.CellTemplate>
+          </modus:DataGridColumn>
+          <!-- Boolean column -->
+          <modus:BooleanColumn Title="Qualified"
```

Comment:
Can you also add `*` and a `fixed width` value? I think we can have fixed value fo the boolean column and the remaining space should be taken by the name column

Reply:
Done